cpp/log/cert.cc

```diff
@@ -1151,27 +1156,27 @@ Cert::Status TbsCertificate::CopyIssuerFrom(const Cert& from) {
   X509_NAME* ca_name = X509_get_issuer_name(from.x509_);
   if (!ca_name) {
     LOG(WARNING) << "Issuer certificate has NULL name";
-    return Cert::FALSE;
+    return util::Status(Code::FAILED_PRECONDITION, "Issuer cert has NULL name");
```

Comment:
This one loses the ability to distinguish that error, but from what I can see, nobody care, this function could even just return a plain `bool`. :stuck_out_tongue: